### ISSUE TITLE
Fix Multi-GPU code

### ISSUE THREAD

**Issue by neale** [neale/Adversarial-Autoencoder#4]
Pytorch merged a patch which was halting development of multi-GPU training code.
This was keeping v1.0 from being fully functional. 

Should validate that we don't get a loss explosion with gradient penalty (double backward) on multi-GPU anymore.
